docs/topics/multiplatform/multiplatform-direct-integration.md

```diff
@@ -1,11 +1,20 @@
 [//]: # (title: Direct integration)
 
+<tldr>
+   This is a local integration method that can work for you if:<br/>
+
+   * You do not import CocoaPods dependencies in your Kotlin Multiplatform project.
+   * You've already set up a Kotlin Multiplatform project targeting iOS on your local machine.<br/>
```

Comment:
This may be nitpicky, but I would change the order of the bullets here: first, the condition about the project as a whole, then the condition about dependencies in that project.

Reply:
makes sense, reordered in all the methods